src/Header.tsx

```diff
@@ -150,15 +150,17 @@ export const RoomHeaderInfo: FC<RoomHeaderInfoProps> = ({
         </Heading>
         <EncryptionLock encrypted={encrypted} />
       </div>
-      {participantCount > 0 && (
+      {(participantCount ?? 0) > 0 && (
         <div className={styles.participantsLine}>
           <UserProfileIcon
             width={20}
             height={20}
             aria-label={t("header_participants_label")}
           />
           <Text as="span" size="sm" weight="medium">
-            {t("participant_count", { count: participantCount })}
+            {participantCount
+              ? t("participant_count", { count: participantCount })
+              : t("participant_count_unkonwn")}
```

Comment:
Why would the "unknown" text ever be triggered? Won't it just be hidden due to the condition a few lines above? (I do think that hiding it is the better UX option here, for the record)